nutanix/config.go

```diff
@@ -55,11 +56,16 @@ func (c *Config) Client() (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	fcClient, err := fc.NewFoundationCentralClient(configCreds)
+	if err != nil {
+		return nil, err
+	}
 	return &Client{
 		WaitTimeout:         c.WaitTimeout,
 		API:                 v3Client,
 		KarbonAPI:           karbonClient,
 		FoundationClientAPI: foundationClient,
+		FC:                  fcClient,
```

Comment:
```suggestion
		FoundationCentral:                  fcClient,
```
Please use full name here